program-log/src/wrapper.rs

```diff
@@ -0,0 +1,62 @@
+//! Logging wrapper for Solana syscalls.
+
+#[cfg(not(any(target_os = "solana", target_arch = "bpf")))]
+use core::hint::black_box;
+#[cfg(any(target_os = "solana", target_arch = "bpf"))]
+use solana_define_syscall::definitions::{sol_log_, sol_log_64_, sol_log_compute_units_};
+
+/// Print a string to the log.
+#[inline(always)]
+pub fn sol_log(message: &str) {
```

Comment:
We could consider dropping the `sol_*` prefix.

Reply:
That's a good point, since these are new functions, let's name them however we want!

I looked around at other rust logging libraries for some standards -- most of them integrate with the standard `log` library, which uses the `info!` / `error!` / `debug!` macros.

I haven't seen too many functions called `log`, but it might be ok in our case for two reasons:
* we don't have log levels, so just `log` makes sense
* we don't want to encourage string formatting, so it shouldn't be a macro

so maybe we have `log`, `log_u64`, log_data`, `log_compute_units`. What do you think?